### DISCUSSION TITLE
Access to the benchmark results bucket is denied

### DISCUSSION THREAD

**Discussion by nikpivkin** [open-policy-agent/community#579] [OPA and Rego]
The documentation [says](https://www.openpolicyagent.org/docs/latest/contrib-development/#benchmarks) that benchmark results are available in the [bucket](https://opa-benchmark-results.s3.amazonaws.com/), but access is denied. Is this outdated information?

**Comment by ashutosh-narkar** [comment]
Hello @nikpivkin yes that's outdated. We run benchmark tests as part of our [Nightly](https://github.com/open-policy-agent/opa/actions/workflows/nightly.yaml) workflow. For example, you can see results from yesterday's run [here](https://github.com/open-policy-agent/opa/actions/runs/8734403047/job/23965011851#step:3:1).

**Comment by nikpivkin** [comment]
@ashutosh-narkar Maybe we should convert the discussion to an issue to fix the documentation?

**Reply by ashutosh-narkar** [reply]
If you'd like to create a quick PR that would be great. Otherwise please create an issue. Thanks!
